### DISCUSSION TITLE
prisma difficulty with accelerate

### DISCUSSION THREAD

**Discussion by Marvyeddy** [prisma/prisma#24805] [Q&A]
### Bug description

why am i requested to use prisma accelerate it was just a test auth app only

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by Malekzie** [comment]
Its more just advertisement to use it. you don't have to

**Comment by ludralph** [comment]
Hi @Marvyeddy 

Could you please provide additional context? Did you encounter the recommendation while executing Prisma commands in the CLI? If so, this is part of our efforts to promote the Accelerate feature to our users.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
